python/feathub/feature_tables/sinks/redis_sink.py

```diff
@@ -29,6 +34,7 @@ def __init__(
         password: str = None,
         db_num: int = 0,
         namespace: str = "default",
+        key_expr: str = DEFAULT_KEY_EXPR,
```

Comment:
It seems better to explicitly use `CONCAT_WS(":", __NAMESPACE__, __KEYS__, __FEATURE_NAME__) ` here so that developer can easily understand the default value.